newIDE/app/src/ProjectManager/index.js

```diff
@@ -700,6 +700,11 @@ export default class ProjectManager extends React.Component<Props, State> {
             autoGenerateNestedIndicator={!forceOpen}
             renderNestedItems={() =>
               filterProjectItemsList(enumerateLayouts(project), searchText)
+                .sort((layout1: gdLayout,layout2: gdLayout) => {
```

Comment:
Unfortunately it's important to show the order to the user because the order is important (first scene is the one launched at the beginning of the game). Can we avoid doing this for now?

Also later we may want to add tags of folders to organize scenes :)

Reply:
@4ian changes have been done